MHM/history/pops/1946.1.1/Denmark.txt

```diff
@@ -1,160 +1,160 @@
 #Nuuk - Kujalleq (11000/2750 POPS)
 251 = {
 	officers = {
-		culture = inuit
+		culture = native_american_minor
 		religion =  protestant 
 		size = 6
 	 } 
 	officers = {
-		culture = inuit
+		culture = native_american_minor
 		religion = secularism
 		size = 1
 	 } 
 	soldiers = {
-		culture = inuit
+		culture = native_american_minor
 		religion =  protestant 
 		size = 27
 	 } 
 	soldiers = {
-		culture = inuit
+		culture = native_american_minor
 		religion = secularism
 		size = 5
 	 } 
 	aristocrats = {
-		culture = inuit
+		culture = native_american_minor
 		religion =  protestant 
 		size = 92
 	 } 
 	aristocrats = {
-		culture = inuit
+		culture = native_american_minor
 		religion = secularism
 		size = 17
 	 } 
 	bureaucrats = {
-		culture = inuit
+		culture = native_american_minor
 		religion =  protestant 
 		size = 4
 	 } 
 	capitalists = {
-		culture = inuit
+		culture = native_american_minor
 		religion =  protestant 
 		size = 10
 	 } 
 	capitalists = {
-		culture = inuit
+		culture = native_american_minor
 		religion = secularism
 		size = 2
 	 } 
 	clergymen = {
-		culture = inuit
+		culture = native_american_minor
 		religion =  protestant 
 		size = 22
 	 } 
 	clergymen = {
-		culture = inuit
+		culture = native_american_minor
 		religion = secularism
 		size = 4
 	 } 
 	artisans = {
-		culture = inuit
+		culture = native_american_minor
 		religion =  protestant 
 		size = 69
 	 } 
 	artisans = {
-		culture = inuit
+		culture = native_american_minor
 		religion = secularism
 		size = 12
 	 } 
 	farmers = {
-		culture = inuit
+		culture = native_american_minor
 		religion =  protestant 
 		size = 1751
 	 } 
 	farmers = {
-		culture = inuit
+		culture = native_american_minor
```

Comment:
change this specific entry to Danish with Protestantism religion, and make all the non-farmer pops (aristocrats, bureaucrats, soldiers, officers, etc.) Danish too, and do the same for the other Greenlandic province.

Reply:
This will reflect the situation in Greenland where Inuit are still dominant culture but Danes have a presence instead of Greenland being 100% Inuit